profiler/src/main/java/com/splunk/opentelemetry/logs/LogsExporter.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.logs;
+
+import java.util.List;
+
+/** Sends a list of LogEntry objects somewhere else, often to the OpenTelemetry collector. */
+public interface LogsExporter {
+  void export(List<LogEntry> logs);
```

Comment:
should this match the java SDK and return `CompletableResultCode`?

Reply:
Yeah probably.  It's all stubbed for now, so maybe when implementing the actual impl would be a good time to make that change?